.infrastructure/helm-chart/todoapp/templates/pvc.yml

```diff
@@ -0,0 +1,13 @@
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Chart.Name }}-pvc
+  namespace: {{ .Values.todoapp.namespace }}
```

Comment:
Ensure that the `todoapp.namespace` value is defined in the `values.yaml` file. This is crucial for the correct deployment of the PersistentVolumeClaim in the specified namespace.